components/ImprovedSearch/CustomClearRefinement.tsx

```diff
@@ -0,0 +1,41 @@
+import { Button } from "@equinor/eds-core-react"
+import { tokens } from "@equinor/eds-tokens"
+import React from "react"
+import { useClearRefinements, UseClearRefinementsProps } from "react-instantsearch-hooks-web"
+import { useIntl } from "react-intl"
+import styled from "styled-components"
+
+const ButtonNew = styled(Button)`
+  color: ${tokens.colors.text.static_icons__default.hex};
+  background: ${tokens.colors.infographic.primary__moss_green_13.hex};
+  border-width: 0;
+  font-weight: 14pt;
+  padding-left: 8.15rem;
+  :hover {
+    color: ${tokens.colors.text.static_icons__default.hex};
+    background: ${tokens.colors.infographic.primary__moss_green_13.hex};
+    border-width: 0;
+  }
+`
+
+const ButtonDisabled = styled(Button)`
+  color: ${tokens.colors.text.static_icons__tertiary.hex};
+  background: ${tokens.colors.infographic.primary__moss_green_13.hex};
+  border-width: 0;
```

Comment:
Same comments as above.

And also a lot of duplicate code here. A better approach would be one of the following:
- Use styled components prop to send in isDisabled to the disabled button and [use conditional styles](https://styled-components.com/docs/basics#adapting-based-on-props)
- Use conditional styles with CSS variable to the component like[ I did here](https://github.com/equinor/data-marketplace/blob/main/htmlParsing/portableText/components/Image.tsx). This might be closer to the Envis approach tho, but just for the record
- [Extend styles with styled components](https://styled-components.com/docs/basics#extending-styles)